tools/restatectl/src/commands/metadata_server/nodes.rs

```diff
@@ -75,51 +80,48 @@ async fn remove_node(
 ) -> anyhow::Result<()> {
     let nodes_configuration = connection_info.get_nodes_configuration().await?;
 
+    let mut metadata_node_set = nodes_configuration
+        .iter_role(Role::MetadataServer)
```

Comment:
should we filter in metadata server state here?

Reply:
Yup, good idea.